core/app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,9 @@
         <%= render "/refinery/header" -%>
       </header>
       <section id="page">
+        <div id="flash_container">
+          <%= render '/refinery/message' %>
+        </div>
```

Comment:
Unfortunately, I am very against adding the flash container onto the application layout.  I feel that this may lead to flash notices bleeding through to users. Is it possible to only have this for the preview controller?